### ISSUE TITLE
Bitcoin Core Does not start

### ISSUE THREAD

**Issue by XxXGyverXxX** [bitcoin/bitcoin#17960]
I attempted to import a private key to a new wallet.  After the rescan process i received a message of "null".
When i attempted to restart the program, the application it gives the following error message,
"Error: Cannot parse configuration file: parse error on line 1: importprivkey <My private key is in these brackets>"

I expected the application to run.

![Screen Shot 2020-01-18 at 6 56 35 PM](https://user-images.githubusercontent.com/35384463/72672185-830b2200-3a24-11ea-8168-040eae797c4b.png)

The error can be reproduced 100% of the time when attempting to open the application

The version of Bitcoin Core is 0.19.0.1.  The version was downloaded from https://bitcoin.org/en/bitcoin-core/.

Macbook Pro 15-inch mid 2012 -  MacOS Mojave V10.14.6

I am sorry if this issue is a simple one.  I am relatively new to using this application.  I would appreciate any help possible.

**Comment by achow101** [comment]
`importprivkey` is a RPC command, not an configuration option. Remove it from your bitcoin.conf file.

**Comment by XxXGyverXxX** [comment]
Achow101

Thanks for the rapid response.  That was what i was looking to do, however, i cannot find the bitcoin.conf file.  
I know it should be, however, the file dosent exist as expected.

**Comment by achow101** [comment]
What exactly did you do when trying to use `importprivkey`? What files did you modify? What terminal commands did you try? How are you starting Bitcoin Core?

**Comment by jeandudey** [comment]
More importantly, where did you get the Bitcoin Core?

**Comment by XxXGyverXxX** [comment]
to answer both questions in the same post:

achow101 - What exactly did you do when trying to use importprivkey? What files did you modify? What terminal commands did you try? How are you starting Bitcoin Core?
--
I went to the console and entered importprivkey wroteprivatekeyhere
I modified nothing else
the terminal command used was importprivkey
I have attempted to start bitcoin core by clicking on the icon in both the applications folder as well as in my taskbar located at the bottom of the screen.


______________________
More importantly, where did you get the Bitcoin Core?
--
the version and download location is 0.19.0.1. The version was downloaded from https://bitcoin.org/en/bitcoin-core/
